examples/solidjs-live-cursors/.gitignore

```
@@ -0,0 +1,11 @@
+.DS_Store
+node_modules
+.env
+.env.*
+!.env.example
+*.tsbuildinfo
+.vercel
+.svelte-kit
+
+dist
+.idea
```

```suggestion
.vercel
dist
```

Regarding the `.idea` folder I think it'd make more sense as a local setting since it references a specific editor.